### ISSUE TITLE
Issue about the CLIP benchmark

### ISSUE THREAD

**Issue by HarborYuan** [UCSC-VLAA/Recap-DataComp-1B#4]
Hi @ImKeTT @xhl-video,

Thanks for your great work. I noticed that the ViT-L/14 in the [OpenCLIP](https://github.com/mlfoundations/open_clip) has a 79.2% accuracy on ImageNet with the DataComp-1B training datasets. Is it the same as `p=1` in the paper?

Thanks for your help.

**Comment by xhl-video** [comment]
I appreciate your interest! Yes, they are the same dataset. The performance difference is primarily due to the shorter training period in our paper (2.56B samples vs. 12.8B samples). Our training recipe mainly follows [CLIPA](https://github.com/UCSC-VLAA/CLIPA/tree/master/clipa_jax). You can find more model performance details using the original DataComp-1B dataset  Please stay tuned for our release of a stronger Recap-CLIP model.

**Comment by HarborYuan** [comment]
Hi @xhl-video ,

Thanks for your quick response. It resolves my question. I am looking forward to the stronger Recap-CLIP model.
